packages/gatsby/static/configuration/manifest.json

```diff
@@ -285,56 +258,60 @@
           "type": "string",
           "enum": ["public", "restricted"],
-          "default": "public",
-          "$comment": "{ \"anchor\": \"publishConfig.access\" }"
+          "examples": ["public"]
         },
         "bin": {
           "description": "If present, the top-level `bin` field from the manifest will be set to this new value before the package is packed to be shipped to remote registries. This won't modify the real manifest, just the one stored within the tarball.",
           "type": "string",
           "format": "uri-reference",
-          "default": "./build/bin.js",
-          "$comment": "{ \"anchor\": \"publishConfig.bin\" }"
+          "examples": ["./build/bin.js"]
         },
         "browser": {
           "description": "Same principle as the `publishConfig.bin` property; this value will be used instead of the top-level `browser` field when generating the workspace tarball.",
           "type": "string",
           "format": "uri-reference",
-          "default": "./build/browser.js",
-          "$comment": "{ \"anchor\": \"publishConfig.browser\" }"
+          "examples": ["./build/browser.js"]
         },
         "executableFiles": {
           "description": "By default, for portability reasons, no files except those listed in the bin field will be marked as executable in the resulting package archive. The executableFiles field lets you declare additional fields that must have the executable flag (+x) set even if they aren't directly accessible through the bin field.",
           "type": "array",
           "items": {
             "type": "string",
             "format": "uri-reference"
-          }
+          },
+          "exampleItems": ["./dist/shim.js"]
         },
         "main": {
           "$ref": "#/properties/main",
-          "$comment": "{ \"anchor\": \"publishConfig.main\", \"overrides\": { \"description\": \"Same principle as the `publishConfig.bin` property; this value will be used instead of the top-level `main` field when generating the workspace tarball.\", \"default\": \"./build/index.js\" } }"
+          "overrides": {
+            "description": "Same principle as the `publishConfig.bin` property; this value will be used instead of the top-level `main` field when generating the workspace tarball.",
+            "examples": ["./build/index.js"]
+          }
         },
         "module": {
           "$ref": "#/properties/module",
-          "$comment": "{ \"anchor\": \"publishConfig.module\", \"overrides\": { \"description\": \"Same principle as the `publishConfig.bin` property; this value will be used instead of the top-level `module` field when generating the workspace tarball.\", \"default\": \"./build/index.mjs\" } }"
+          "overrides": {
```

Comment:
Is that an extension too?

Reply:
Yep, this was an extension that I've added because I didn't want to have to pass a customizer to the `merge` function to later know what are the originals and what are the overrides. I've since realized that other interpreters would pick up overrides at the same level as the ref, so I've refactored it to use a customizer.